moccasin/config.py

```diff
@@ -174,9 +249,10 @@ def _fetch_contracts_from_db(
             chain_id = to_hex(self.chain_id)
         else:
             chain_id = to_hex(chain_id)
-        return db._get_deployments_from_sql(
-            GET_CONTRACT_SQL.format(field_names, limit_str), (contract_name, chain_id)
+        final_sql = GET_CONTRACT_SQL.format(
+            field_names, "'" + contract_name + "'", chain_id, limit_str
```

Comment:
this can lead to SQL injection (the contract name should be safe, but it can still cause errors when the name has `'`)

Reply:
Yeah.... I think I fixed this in the most recent commit.